doc/ref_model/chapters/chapter08.md

```diff
@@ -98,8 +98,63 @@ The Cloud Services and the Cloud Resources Brokers provide value-added services
+To realise a federated cloud requires the definition and agreement on a set of APIs. These APIs should allow each of the parties to interact cooperatively and need to cover the management layer: business management and service operations interactions; as well as the data plane, customer and user, transactions and conversational interfaces.
+
+As outlined in "Figure 8-3 - Conceptual Architecture of a Telco Operator Platform" above the exposure point for the Management Interactions is the "Cloud Service Broker" and the "Cloud Resource Broker". The set of interactions that these interface points need to provide are defined by the "Multi-Cloud Interaction Model". This provides a taxonomy for the interactions and is illustrated below:
+
+<p align="center"><img src="../figures/rm-chap8-multi-cloud-interactions-02.png" alt="Multi-Cloud Interactions Model"></p>
+<p align="center"><b>Figure 8-4:</b> Multi-Cloud Interactions Model</p>
+
+The model defines the following core roles:
+- Communications Service Provider (CSP) - is the party responsible for providing end user service to their customer
+- Customer / User - are the parties that use the service (User) and establishes the business agreement for the service provision (Customer). For retail services the customer and user are the same party, while for enterprise services the Enterprise is the Customer (responsible for the business agreement) and its representatives are the Users.
+- Cloud Providers - are the parties providing the cloud services. These services could be any XaaS service. It could be that a CSP has an agreement with a SaaS Cloud, which in turn uses an IaaS Cloud Provider to deliver their service.
+
+The set of high level interactions cover:
+- Manage Account - covering Account, Users, Subscription, Billing
+- Manage Connectivity - Public or Private Network, VPN Configuration, CSP Edge / Cloud Connection Configuration, Connection Security Profile
+- Manage Resource - Resource Pool Management, VM / VNF Management (CPU, Memory, Storage, Network), Image Repository Management, Storage Management, VNF / CNF LCM, Monitor Resources
+- Manage App/VNF - Image / Container / Registry Management, Deploy/Configure/Scale/Start/Stop App/VNF, Monitor App/VNFs
+- Transactions / Conversations - Use Communications Services, Use Edge Applications Services, Use Cloud Services
+
+<a name="8.2.4.2"></a>
+### 8.2.4.2 Stereo-Typical Scenarios
+A set of stereo-typical interactions cases are illustrated for the cases of a Simple Infrastructure-as-a-Service (IaaS) and Software-as-a-Service (SaaS), where deployment is on a Cloud Provider's centralised sites and/or Edge sites. The scenarios help highlight needs for the Cloud Service Broker and Cloud Resources Broker (as per Figure 8.3) and hence extent of orchestration required to manage the interactions.
+
+<p align="center"><img src="../figures/rm-chap8-multi-cloud-interactions-simple-stereo-types-02.png" alt="Simple Stereo-Typical Interactions"></p>
```

Comment:
Please share the source of the figure in the [figures folder](https://github.com/cntt-n/CNTT/tree/master/doc/ref_model/figures).